packages/esm-messages-app/src/api/api-remote.ts

```diff
@@ -0,0 +1,18 @@
+import { openmrsFetch } from '@openmrs/esm-framework';
+async function postData(url: string, data = {}, ac = new AbortController()) {
```

Comment:
I find it weird that this is called "postData" when the actual verb is `PUT`, especially since this seems _really_ generic.